src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,114 @@ This new object is an array of example objects that can successfully POST agains
+
+**Follow the steps as outlined below**
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. One of the possible enums for for the genetic modifications object was "validation" up until schema version 6. As a part of schema version upate to 7, "validation" was removed and instead the term "characterization" was added.
+
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+        # Changing validation to characterization as a list of enums within the purpose property:
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. For example if the original schema version for the genetic modification object being modified was "6", change it to "7" (6->7): 
         
         "schema_version": {
-            "default": "2"
+            "default": "7"
         }
 
-2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example upgrades the object from schema version (2->3):
+2. In the **upgrade** directory add an ```upgrade_step``` to an existing/new python file named after the object. An example to the upgrade step is shown below. Continuing with our example, after the upgrade all the existing objects with that had "purpose" specified to be "validation" must now be changed to "characterization". And since the schema is changing from version 6 to 7 the def must specify this (6->7):
```

Comment:
I would say "to a python file named after the object (create new if there is no such a file in the **upgrade** directory" probably worth mentioning that for some objects the upgrade happens in the abstract class - like dataset for the experiment.